### ISSUE TITLE
Avoiding duplicated efforts

### ISSUE THREAD

**Issue by rsalvaterra** [gjedeer/androidwisprclient#7]
Hi there! You may want to take a look at this:

https://github.com/rsalvaterra/Fon

I found your repository be mere chance. I have been hacking on this project mostly for fun and things, well, got a bit out of hand. :)

**Comment by gjedeer** [comment]
I did not have a chance to compile your code (but I will when I do),
but it looks much better than the original program. 

When you feel that your code is mature enough and it's accepted to
F-Droid, I'll ask the F-Droid maintainers to remove my fork - given
that yours works well on my phone, of course.

**Comment by alanthehat** [comment]
Please don't remove this from F-Droid as alternatives are always a good thing. 
Whether or not rsalvaterra's version is absolutely perfect on your devices or mine there are going to be devices that don't work well with it.

**Comment by rsalvaterra** [comment]
If it doesn't work, it's a bug. I'm deliberately building for API level 3 and up with no support libraries (hence the deprecation warnings you may see in my code, though I tried to limit the classes that contain them). I personally tested it on Android 2.1 (my very old ZTE Racer, for testing), 4.1.2 (an Onda tablet, which I use for development), 4.3 (my mom's Huawei Y530), 4.4.4 (my OnePlus One) and 5.0.1 (a close friend's Nexus 4).

**Comment by alanthehat** [comment]
Oh, very definitely there are buggy devices -- I have a generic Chinese phone!
And I really appreciate that you're building for & testing against a wide range of real devices.

**Comment by rsalvaterra** [comment]
Indeed, I've been personally bitten by devices violating the Android API documentation (with methods returning null when they should never do so, for example).
